### DISCUSSION TITLE
gold dungeon

### DISCUSSION THREAD

**Discussion by BoomExplode** [DiscordDungeons/Bugs#984] [Ideas]
add a new dungeon in dangerou cavern. instead of having gemstone, they have gold ore, (either its gonna be very rare or the enemy is gonna be very hard cuz they need a reason to actually enter the dungeon themselves), and either way the enemy wont start at ur level (1.5 maybe?) and theres more chance for mimic to spawn from chest. and add something to make the dungeon more appealing cuz people probably would still choose mining over dungeoning

**Comment by Feathers8** [comment]
Something more appealing than gold

**Reply by BoomExplode** [reply]
gold ore :/

**Reply by Feathers8** [reply]
Nah. Gold isn't much of a problem compared to gems
It's not tedious to grind gold, but it sure is for gems

**Reply by BoomExplode** [reply]
what about both?

**Reply by Feathers8** [reply]
Still may not be a good enough incentive to go through a harder dungeon

I was thinking it could house mobs possessing exclusive drops but... dunno what cool new craftables they could add lol. Maybe an alternative recipe for the gold ring? For those who don't have the letter
